openfisca-web-api/input-output-data.md

```diff
@@ -1,104 +1,294 @@
-# Input and output data
+# Describing a situation in OpenFisca
 
-## Values by period
+> All the example provided here are from the [country package template](https://github.com/openfisca/country-template).
 
-In OpenFisca, the value of a variable is always defined for a specific period.
+When using the API `/calculate` endpoint, you will need to send information to the API concerning :
```

Comment:
Starting with a description of what a situation is? The user might ask why we are speaking about `/calculate` in a page about situations.